doc/source/Resources/pyaedt_settings.yaml

```diff
@@ -0,0 +1,117 @@
+# This file contains the settings used by default to set the PyAEDT environment variables and
+# global settings. If you want to have a different behavior you have to copy / paste this file
+# into your PersonalLib directory and restart your workflow.
```

Comment:
@SMoraisAnsys Could be in any location right? PersonalLib is only one option right?

Reply:
Yeah, I'll update the doc